api3/global/openapi.gen.yaml

```diff
@@ -603,7 +603,6 @@ components:
       - operation
       - requestsCount
       - findings
-      - highestSeverity
```

Comment:
I think it should be required in  `FuzzingReportOperation` unless `findings` can be empty (and then `findings` should not be required but I don't think this is the case here)

Reply:
We can have no findings, so I'll make it optional.